src/User32.Desktop/User32.Helpers.cs

```diff
@@ -47,5 +47,24 @@ public static unsafe string GetClassName(IntPtr hWnd)
 
             return new string(className, 0, count);
         }
+
+        /// <summary>
+        /// Retrieves from the clipboard the name of the specified registered format.
+        /// </summary>
+        /// <param name = "format">The type of format to be retrieved. This parameter must not specify any of the predefined clipboard formats.</param>
+        /// <returns>The format name string.</returns>
+        /// <exception cref="Win32Exception">Thrown when an error occurs.</exception>
+        public static unsafe string GetClipboardFormatName(int format)
+        {
+            const int bufferSize = 256;
```

Comment:
> The system uses atom tables that are not directly accessible to applications. However, the application uses these atoms when calling a variety of functions. For example, registered clipboard formats are stored in an internal atom table used by the system. An application adds atoms to this atom table using the RegisterClipboardFormat function. Also, registered classes are stored in an internal atom table used by the system. An application adds atoms to this atom table using the RegisterClass or RegisterClassEx function.
From [MSDN: About Atom Tables](https://msdn.microsoft.com/en-us/library/windows/desktop/ms649053(v=vs.85).aspx)

If we add this knowledge of the internals of this function to the limits we get from registering an Atom with [GolbalAddAtom function](https://msdn.microsoft.com/en-us/library/windows/desktop/ms649060(v=vs.85).aspx) we see that the limit of the null-terminated string is 255 characters, I placed an extra for the terminating null 😉.